examples/bevy_wgpu_render.rs

```diff
@@ -1,54 +1,33 @@
-#[cfg(feature = "bevy_wgpu")]
-use bevy::prelude::*;
-#[cfg(feature = "bevy_wgpu")]
-use shocovox_rs::{octree::raytracing::OctreeViewMaterial, octree::V3c};
+use bevy::{prelude::*, window::WindowPlugin};
+use shocovox_rs::octree::{
+    raytracing::{ShocoVoxRenderPlugin, ShocoVoxViewingGlass, Viewport},
+    V3c,
+};
+
+const RESOLUTION: [u32; 2] = [320, 240];
 
-#[cfg(feature = "bevy_wgpu")]
 fn main() {
     App::new()
+        .insert_resource(ClearColor(Color::BLACK))
         .add_plugins((
-            DefaultPlugins,
-            MaterialPlugin::<OctreeViewMaterial>::default(),
+            DefaultPlugins.set(WindowPlugin::default()),
+            ShocoVoxRenderPlugin {
+                resolution: RESOLUTION,
```

Comment:
The best case scenario would be to have `ShocoVoxRenderPipeline` store the resolution, but then the output textures would need to be created automatically  as well, I'll try to dig into it

Reply:
The only way I know to do this would be to add a Resource 

```
#[derive(Resource)]
pub(crate) struct ShocoVoxRenderResolution(pub(crate) [u32; 2]);
```

And then in the plugin add a modified version of `create_output_texture` to startup:

```
app.add_systems(Startup, create_ouput_texture2);
```

But this would only make the creation of the image handle automatic, it would not connect the viewing glass to this image in any way that I can see. 

I think resolving this would be at the moment be more energy, than its worth, especially given the API maturity. In the future I'd like to give the opportunity to have multiple views of the same data, possibly with multiple resolutions. For that the API needs to be adapted as well. 


If you think this is worthy of an issue of its own, then we can open up an issue for it. But I don't think this should be solved in this PR.